sdk/python/touca/cli/_results.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from argparse import ArgumentParser
+from pathlib import Path
+from shutil import rmtree
+
+from touca._options import find_home_path
+from touca._printer import print_table
+from touca.cli._common import Operation, ResultsTree, invalid_subcommand
+
+logger = logging.Logger("touca.cli.results")
+
+
+class Results(Operation):
+    name = "results"
+    help = "Show suite results"
+
+    def __init__(self, options: dict):
+        self.__options = options
+
+    @classmethod
+    def parser(self, parser: ArgumentParser):
+        parsers = parser.add_subparsers(dest="subcommand")
+        results_dir = find_home_path().joinpath("results")
+
+        parsers_list = parsers.add_parser(
+            "list", help="show a summary of test results"
+        )  # noqa: E501
+        parsers_remove = parsers.add_parser(
+            "remove", help="remove test results"
+        )  # noqa: E501
+
+        for pars in (parsers_list, parsers_remove):
```

Comment:
This is great and I know the description used `list` and remove` so your impl is consistent with the specified requirements but we recently decided to use the short-hand `ls` and `rm` that are more developer friendly. So I would suggest we use `ls` instead of `list` and `rm` instead of `remove` as arguments.
```suggestion
        parser_ls = parsers.add_parser("ls", help="list local touca archive files")
        parser_rm = parsers.add_parser("rm", help="remove local touca archive files")
        results_dir = find_home_path().joinpath("results")
        for parser in [parser_ls, parser_rm]:
```